guides/rebalancer_user_guide.md

```diff
@@ -0,0 +1,50 @@
+# Rebalancer User Guide
+
+## Introduction
+
+The rebalancer is a Valence service designed for Valence account holders, including users and DAOs, to adjust their portfolio or treasury to a desired target. During each rebalance cycle, users determine which assets need to be bought or sold to reach their target. Assets marked for sale are then auctioned off, allowing anyone to place bids on them.
+
+## How to use it
+
+### DAODAO
+
+DAODAO offers a user interface that allows users to create an account, register it with the rebalancer using their preferred parameters, view the account's status, and manage it effectively. Here are the steps:
+
+1. Create an account and register to the rebalancer.
+
+   The first thing we have to do is create an account and register to the rebalancer, this is easily done by creating a Rebalancer proposal.
+
+   ![Rebalancer proposal](./img/rebalancer/proposal_before.png)
+
+   By clicking on `Configure Rebalancer`, a modal will open with two options: `Create Rebalancer Account` and `Configure Rebalancer`. The `Create Rebalancer Account` option allows you to set up an account and fund it with the specified initial balances, along with the registration fee for the rebalancer (currently 1 NTRN). Example:
+
+   ![Create rebalancer account](./img/rebalancer/create_account.png)
+
+   In the example above, we are creating an account and funding it with 11 NTRN: 10 NTRN will be used as the account balance and 1 NTRN as the registration fee.
+
+   NOTE: Not all tokens are supported by the rebalancer. The rebalancer has a set of whitelisted denoms and optional minimum balances for each denom that the account must hold to be considered for as `Base token`. Therefore, when creating an account via DAODAO, the token used as `Base token` must be funded with at least its minimum balance. For example, at the time of this proposal, the minimum balance for NTRN and ATOM is 10. If you attempt to create a rebalancer account and fund it with 9 NTRN instead of 10 with NTRN as `Base token`, the proposal will not be created, and an error message regarding minimum balances will appear. On the other hand, if you attempt to create a rebalancer account and fund it with 10 NTRN and 2 ATOM with NTRN as `Base token`, the proposal will be created successfully, even though the minimum balance for ATOM is not met.
```

Comment:
```suggestion
**NOTE:** Not all tokens are supported by the Rebalancer. The Rebalancer has a set of whitelisted denoms that it supports. Additionally, each account must be configured with a `Base token`. The account value is denominated in this token. Each account is required to maintain a minimum balance (specified differently for each `Base token`). The following example illustrates this concept:
> An account configures their `Base token` to be NTRN. The minimum balance required for this account is 10 NTRN. This does not mean that the account must possess at least 10 NTRN tokens, simply that 10 NTRN worth of tokens in the whitelisted denoms must be present in the account. The account can be created with 5 NTRN token and 5 ATOM tokens, for example, as long as the 5 ATOM + 5 NTRN are valued greater than 10 NTRN tokens which is the minimum balance requirement. 

Minimum balances are specified per `Base token` so it is possible that the minimum balance requirement is met under one `Base token` but not under another. 
```

Reply:
Yeah was not convinced with my version, I think because I was confused myself during first test iteration, this is def much better, thanks.

Interesting, I didn't know the second part myself, good to know this (tested it also just in case)